core/templates/dev/head/pages/admin/Admin.js

```diff
@@ -258,6 +258,50 @@ oppia.controller('Admin', [
       });
     };
 
+    $scope.reloadAllCollections = function() {
```

Comment:
@kahkhang I agree with @seanlip 's suggestion to use the existing the button to reload the collection.